### ISSUE TITLE
[Bug]: Cannot find module '@fluentui/react' or its corresponding type declarations.

### ISSUE THREAD

**Issue by HarminderSethi** [microsoft/fluentui#24467]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.22000
    CPU: (8) x64 Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz
    Memory: 5.92 GB / 15.76 GB
  Browsers:
    Edge: Spartan (44.22000.120.0), Chromium (104.0.1293.63)
    Internet Explorer: 11.0.22000.120
```

### Are you reporting Accessibility issue?

no

### Reproduction

N/A

## Actual Behavior
After updating the @fluentui/react with 8.90.2 version, I started getting error "Cannot find module '@fluentui/react' or its corresponding type declarations". in SPFx solution(version 1.15.2). version 8.90.1 and 8.90.0 working fine.
![image](https://user-images.githubusercontent.com/17841313/186165114-6c12371a-adba-4f41-a0c4-c91a49cdde60.png)

## Expected Behavior
should work similar to previous release.

### Requested priority

High

### Products/sites affected

N/A

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by raphael10-collab** [comment]
I'm getting the same error : 

    import { Stack } from "@fluentui/react"

Found 62 errors in 9600 ms

    raphy@pc:~/FPlayground$ yarn add --dev @types/@fluentui/react
    yarn add v1.22.19
    [1/4] Resolving packages...
    error Command failed.
    Exit code: 128
    Command: git
    Arguments: ls-remote --tags --heads ssh://git@github.com/fluentui/react.git
    Directory: /home/raphy/FPlayground
    Output:
    git@github.com: Permission denied (publickey).
    fatal: Could not read from remote repository.
    
    Please make sure you have the correct access rights
    and the repository exists.
    info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
    raphy@pc:~/FPlayground$ 

Other Info:

    O.S.: Ubuntu 20.04 Dekstop
    node: v16.15.1
     "@fluentui/react": "^8.90.2"

**Comment by TonyTroeff** [comment]
This is related to #24464

**Comment by micahgodbolt** [comment]
Ok, I spun up a local create react app and am able to repro this issue. I'm looking into it.

**Comment by micahgodbolt** [comment]
We found the bug in our publishing code. We'll have a PR up shortly and release another version.

**Comment by rudymiked** [comment]
This is set as closed, but the errors are still present for me. I have deleted node_modules and have re-installed @FluentUI

Can anyone confirm that this is fixed?

**Comment by TylerDurham** [comment]
I just re-added to my project and tsc ran fine... think it is good now.

**Comment by rudymiked** [comment]
![image](https://user-images.githubusercontent.com/6699164/186256645-644029c1-897c-42e2-a406-28530d0c5851.png)

still see this message after re-installing @fluentui

**Comment by HarminderSethi** [comment]
@rudymiked  version 8.90.2 is marked as deprecated. version 8.90.1 is the latest active version. As confirmed by @micahgodbolt new version with the fix will be released soon.
![image](https://user-images.githubusercontent.com/17841313/186299361-d5446009-4484-40a6-aa85-a9b9eaab1c59.png)

**Comment by markdon** [comment]
Does this effect packages other than @fluentui/react? 
I can't get my node_modules in an existing project back in to a working state as some dependencies of @fluentui/react (like @fluentui/react-hooks) seem to show the same problem. Trying to install an earlier version of @fluentui/react still installs latest versions of dependencies which are broken.

**Comment by deepak-chandani** [comment]
These errors occur on v8.11.x as well (`@fluentui/react@8.11.2`).
I deleted node modules and reinstalled packages (using 'npm install'), but I still receive errors. 

<img width="1127" alt="image" src="https://user-images.githubusercontent.com/15975603/186331798-b0895b05-6c63-48c7-993d-e02c16261cf8.png">

**Comment by jarvan-jiang** [comment]
I cannot make any version of @fluentui/react working as long as it needs install now

**Comment by HarminderSethi** [comment]
Facing same problem while building the project.  Had to switch to office-fabric-ui-react for time being. SPFx version 1.15.2

**Comment by abhiagra** [comment]
This is still not working for us. Please share if anyone has found a solution or workaround.

**Comment by amitShimon1983** [comment]
Hey, have had the same issue since yesterday. My fluent-UI version is  "@fluentui/react": "^8.72.0", my react version 17 everything work on sendbox when build I'm getting the same errors as the thread owner gets Error: Can't resolve '@fluentui/react-hooks' in '/app/node_modules/@fluentui/react/lib/c

**Comment by ag0059985** [comment]
hello @micahgodbolt, I am still facing the same issue, have delete node_modules and package-lock file, and reinstall modules.
but no luck, i have tried with 8.90.2 version of fluentUI

**Comment by ol74eg** [comment]
Same problem with 8.91.0 version

**Comment by khaledfarghaly** [comment]
same with 8.90.2 version

**Comment by AtDH01** [comment]
Also seeing the same when using 8.90.2 @micahgodbolt

**Comment by Front-start** [comment]
The workaround is to use previous version of these packages:

With yarn:
` "resolutions": {
    "@fluentui/react-hooks": "8.6.7",
    "@fluentui/style-utilities": "8.7.7",
    "@fluentui/foundation-legacy": "8.2.15",
    "@fluentui/font-icons-mdl2": "8.4.8",
    "@fluentui/react-focus": "8.8.0",
    "@uifabric/styling**/@fluentui/theme": "1.7.7",
    "@fluentui/react/**/@fluentui/theme": "2.6.12"
  },`
  
  With npm (8.3.0+):
`  "overrides": {
    "@fluentui/react": {
      "@fluentui/react-hooks": "8.6.7",
      "@fluentui/style-utilities": "8.7.7",
      "@fluentui/foundation-legacy": "8.2.15",
      "@fluentui/font-icons-mdl2": "8.4.8",
      "@fluentui/react-focus": "8.8.0",
      "@fluentui/theme": "2.6.12",
      "react-hooks": "8.6.7",
      "style-utilities": "8.7.7",
      "foundation-legacy": "8.2.15",
      "font-icons-mdl2": "8.4.8",
      "react-focus": "8.8.0",
      "theme": "2.6.12"
    },
    "@uifabric/styling": {
      "@fluentui/theme": "1.7.7"
    }
  }`
  
You might need to add another packages to this list. Just check build log for the failing package. Then go to its version list and find one that was released before yesterday.

**Comment by ag0059985** [comment]
thanks @Front-start I have pasted the following packages as dependencies in package.json file, and not getting the error.
```
"@fluentui/react-hooks": "8.6.7",
      "@fluentui/style-utilities": "8.7.7",
      "@fluentui/foundation-legacy": "8.2.15",
      "@fluentui/font-icons-mdl2": "8.4.8",
      "@fluentui/react-focus": "8.8.0",
      "@fluentui/theme": "2.6.12"
```

**Comment by amitShimon1983** [comment]
> thanks @Front-start I have pasted the following packages as dependencies in package.json file, and not getting the error.
> 
> ```
> "@fluentui/react-hooks": "8.6.7",
>       "@fluentui/style-utilities": "8.7.7",
>       "@fluentui/foundation-legacy": "8.2.15",
>       "@fluentui/font-icons-mdl2": "8.4.8",
>       "@fluentui/react-focus": "8.8.0",
>       "@fluentui/theme": "2.6.12"
> ```

Hey, I did as you suggested, but I'm working with yarn workspaces, and it didn't help me, so I had to downgrade to fluent version 7. if someone knows how I can solve it, ill be more then grateful

**Comment by ag0059985** [comment]
> 



> > thanks @Front-start I have pasted the following packages as dependencies in package.json file, and not getting the error.
> > ```
> > "@fluentui/react-hooks": "8.6.7",
> >       "@fluentui/style-utilities": "8.7.7",
> >       "@fluentui/foundation-legacy": "8.2.15",
> >       "@fluentui/font-icons-mdl2": "8.4.8",
> >       "@fluentui/react-focus": "8.8.0",
> >       "@fluentui/theme": "2.6.12"
> > ```
> 
> Hey, I did as you suggested, but I'm working with yarn workspaces, and it didn't help me, so I had to downgrade to fluent version 7. if someone knows how I can solve it, ill be more then grateful

in my case, i am using npm and instead of adding as "overrides", i added these packages as "dependencies".

**Comment by micahgodbolt** [comment]
We just kicked off a new release of all of those affected packages

**Comment by micahgodbolt** [comment]
Just did a fresh install in my CRA, pulling in 8.91.1, and had success.

**Comment by ag0059985** [comment]
> Just did a fresh install in my CRA, pulling in 8.91.1, and had success.

yep, 8.91.1 version has solved the issue.
thanks @micahgodbolt

**Comment by zaiby** [comment]
8.91.1 resolved the issue :)

**Comment by raphael10-collab** [comment]
Solved with 8.91.1 version

Thank you @micahgodbolt
